### ISSUE TITLE
Learning rate and stopping criterion

### ISSUE THREAD

**Issue by matsen** [matsengrp/netam#101]
Training can look like this:

![image](https://github.com/user-attachments/assets/176f503c-66d3-4d90-bcd9-99676ac3bb79)

Is that too big of a learning rate for the green model?

Also, I would have thought that [ReduceLROnPlateau](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html) would have stopped things earlier, but perhaps the upper loss is keeping things going? Should we change the threshold? (I _think_ we'd want to increase the threshold, but it's a sleepless night...)

**Comment by matsen** [comment]
@willdumm Here's a paper that suggest a warm-up then a cool-down http://dx.doi.org/10.1101/2024.05.14.594108

I think it would be worth our while to have a `grid` about this.

**Comment by matsen** [comment]
And here's a paper that suggests smoothly interpolating between unpaired and paired sequences 🤷 https://www.biorxiv.org/content/10.1101/2025.02.27.640641v1
